
syntax = "proto3";

package api;

message GetUserRequest {
    int32 Id = 1;
}

message GetUserResponse {
    string Name = 1;
}

service UserService{
    rpc GetUser (GetUserRequest) returns (GetUserResponse) {}
}